res/css/views/dialogs/_RoomSettingsDialog.scss

```diff
@@ -42,3 +42,11 @@ limitations under the License.
     padding-left: 40px;
     padding-right: 80px;
 }
+
+// show a different AvatarSetting placeholder for RoomProfileSettings which is basically a clone of ProfileSettings
+.mx_RoomSettingsDialog .mx_AvatarSetting_avatar .mx_AvatarSetting_avatarPlaceholder::before {
+    mask: url("$(res)/img/feather-customised/image.svg");
```

Comment:
`avatar_placeholder.svg` or something maybe?

Reply:
Nooooooo

Reply:
Oh I see, it really is an image